profile-diff/style.css

```diff
@@ -0,0 +1,352 @@
+  --dark-blue: #1b1378;
+  --light-aqua: #d4f9f2;
+  --scandal: #e5fcf5;
+  --green-transparent: rgba(0, 255, 0, 0.2);
+  --green-color: green;
+  --red-transparent: rgba(255, 0, 0, 0.145);
+  --white: #ffffff;
+  --black-transparent: #000000a8;
+  --black: #181717;
+  --light-gray: #d9d9d9;
+  --razzmatazz: #df0057;
+  --red-color: red;
+  --gray: #808080;
+  --button-proceed: #008000;
+  --modal-color: #00000048;
+  --black-color: black;
+  --light-gray-color: lightgray;
+  --green10: #e1f9f1;
+  --green500: #19805e;
+  --secondary10: #fff0f6;
+  --secondary600: #b6004e;
+  --medium-gray: #aeaeae;
+  --dark-gray: #737373;
+  --blue-color-heading: #041187;
+  --white-gray: #f2f2f3;
+  --color-red: #ae1820;
+  --color-green: rgba(0, 128, 0, 0.8);
+  --color-warn: rgba(199, 129, 18, 0.8);
+}
+
+*,
+::after,
+::before {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  user-select: none;
+  max-width: 100vw;
+  background-color: #f8fafd;
+}
+
+.header {
+  height: 7.25em;
+  background-color: #233876;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-optical-sizing: auto;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 1.25em;
+  color: var(--white);
+  position: relative;
+}
+
+.back {
+  background-color: transparent;
+  border: none;
+  padding: 0.625em;
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+  position: absolute;
+  left: 10%;
+}
+
+.back__icon {
+  height: 1.6rem;
+}
+
+.container {
+  margin: 0 auto;
+  padding: 2em 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1em;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: x-large;
+  text-align: center;
+  width: 95%;
+  max-width: 992px;
+}
+
+.user_image {
+  border-radius: 37px;
+}
+
+.user_name {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 150%;
+  color: #111928;
+  margin-top: -1em;
+  margin-bottom: 1em;
+}
+
+.expand-control-container {
+  width: 100%;
+  display: flex;
+  justify-content: end;
+  align-items: center;
+  color: #233876;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+}
+
+.expand-control-text {
+  cursor: pointer;
+}
+
+.expand-control-icon {
+  margin-left: 0.5em;
+  font-size: 10px !important;
+}
+
+.reason_heading {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 150%;
+  color: #111928;
+  width: 100%;
+  text-align: left;
+  padding-left: 1.25em;
+  margin-block-end: 0;
+}
+
+.reason_textarea {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  background: #ffffff;
+  box-shadow: 0px 8px 3px rgba(0, 0, 0, 0.01), 0px 5px 3px rgba(0, 0, 0, 0.02),
```

Comment:
- should we create a css variable and use it everywhere we're using this box shadow ?